Cargo.toml

```diff
@@ -1,21 +1,34 @@
 [package]
-name = "socket2"
-version = "0.4.0-dev"
-authors = ["Alex Crichton <alex@alexcrichton.com>"]
-license = "MIT/Apache-2.0"
-readme = "README.md"
-repository = "https://github.com/alexcrichton/socket2-rs"
-homepage = "https://github.com/alexcrichton/socket2-rs"
+name          = "socket2"
+version       = "0.4.0-dev"
+authors       = ["Alex Crichton <alex@alexcrichton.com>"]
+license       = "MIT/Apache-2.0"
+readme        = "README.md"
+repository    = "https://github.com/rust-lang/socket2-rs"
+homepage      = "https://github.com/rust-lang/socket2-rs"
+documentation = "https://docs.rs/socket2"
 description = """
 Utilities for handling networking sockets with a maximal amount of configuration
 possible intended.
 """
-edition = "2018"
+keywords      = ["io", "socket", "network"]
+categories    = ["api-bindings", "network-programming", "web-programming"]
+edition       = "2018"
+include       = [
+  "Cargo.toml",
+  "LICENSE-APACHE",
+  "LICENSE-MIT",
+  "README.md",
+  "src/**/*.rs",
+]
```

Comment:
Once we start adding tests it will make a difference, for example Mio 0.7 is ~80 kB versus Mio 0.6 ~100 kB. Doesn't sound like much, but times that by ~14 million downloads and it safes our hosts and users quite a bit of bandwidth.

Reply:
I see. Wouldn't it be better to use `exclude = ["tests"]` though? Explicit opt-out sounds safer than an explicit opt-in.

Reply:
When packaging (part of `cargo release`) it will run `cargo check` on the to be uploaded package, so it any required files are not included it will fail to release it. With an include we know we know we only include the files we want, with an exclude we need to keep that up-to-date more, e.g. we also have an `.github` dir we would need to exclude. But in the end there is not much of a difference.